### PR TITLE
Add work items doc and add better validation in work items commands

### PR DESCRIPTION
**Pull request by mmanela**
1. Add docs for work item commands
2. Update validation in Update/Create to catch common errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ tfx <command> --help
 * `tfx build` ([builds](docs/builds.md)): Queue, view, and get details for builds
 * `tfx build tasks` ([build tasks](docs/buildtasks.md)): Create, list, upload and delete build tasks
 * `tfx extension` ([extensions](docs/extensions.md)): Package, manage, publisher Team Foundation Server / Team Services extensions
+* `tfx workitem` ([work items](docs/workitems.md)): Create, query and view Work Items.
```
Comment (willsmythe) [inline README.md]: lowercase "work items"

File: docs/workitems.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
+# TFX Work Item commands
+
+Create, query and view [Work Items](https://www.visualstudio.com/en-us/docs/work/backlogs/add-work-items).
+
+## Get started
```
Comment (willsmythe) [inline docs/workitems.md]: Not sure where, but I'd call out (maybe a markdown > block quote) the problem you mentioned yesterday --- error that is shown when users try to use a non-collection level URL.

**Comment by trevorsg** [comment]
Looks good, thanks @mmanela. Feel free to merge when you're ready - I gave you write permissions. I'll publish the new build once you've merged.

**Comment by mmanela** [comment]
Merged. @willsmythe Let me know if there is anything else to fix before @trevorsg makes a new release

**Comment by willsmythe** [comment]
Looks good to me.

**Comment by trevorsg** [comment]
Thanks everyone, I have published the new version.

**Pull request merged**